lib/build/tasks.js

```diff
@@ -71,6 +71,10 @@ function ensureTempDirExists (dir = tempPath) {
 
 function sassExtensions () {
   let fileContents = '$govuk-extensions-url-context: "/extension-assets"; $govuk-prototype-kit-major-version: 13;\n'
+  if (extensions.legacyGovukFrontendFixesNeeded()) {
+    fileContents += '$govuk-global-styles: true;'
+    fileContents += '$govuk-new-link-styles: true;'
```

Comment:
```suggestion
    fileContents += '$govuk-global-styles: true !default;'
    fileContents += '$govuk-new-link-styles: true !default;'
```

As per suggestion from @36degrees https://github.com/alphagov/govuk-frontend/pull/2851#discussion_r985538527